dac/pkg/collection_point.go

```diff
@@ -81,7 +81,7 @@ func (d dacCollectionPoint) SaveFulfillment(fulfillment Fulfillment) error {
 }
 
 func SignFulfillment(fulfillment *Fulfillment, scheme crypto.Scheme) error {
-	signature, err := scheme.Sign([]byte(fulfillment.Cid + base64.StdEncoding.EncodeToString(fulfillment.SessionId) + fulfillment.RequestId + strconv.FormatUint(fulfillment.FulfilledTimestamp, 10)))
+	signature, err := scheme.Sign([]byte(fulfillment.Cid + base64.StdEncoding.EncodeToString([]byte(fulfillment.SessionId)) + fulfillment.RequestId + strconv.FormatUint(fulfillment.FulfilledTimestamp, 10)))
```

Comment:
Given you changed the type of `SessionId` from `[]byte` to `string`, maybe we can just concatenate it here without `EncodeToString` - the same as for `Cid` and `RequestId`? I mean in the result signature will not it be base64 encoded? It will lead to failed sig check on collection point

Reply:
yep, you're right
changed to simple string concatenation